packages/config/config/devices/0x0312/mp21zp.json

```diff
@@ -43,33 +43,15 @@
 		},
 		{
 			"#": "6",
-			"label": "Automating Reporting: Power Interval",
-			"valueSize": 4,
-			"unit": "minutes",
-			"minValue": 1,
-			"maxValue": 54424,
```

Comment:
This seemed like a bug to me. "54424" is "65535" shifted down 1 each decimal digit.